### DISCUSSION TITLE
Beginner set up

### DISCUSSION THREAD

**Discussion by remedy123ate** [transitive-bullshit/nextjs-notion-starter-kit#314] [Q&A]
Dear Travis, 

Thank you so much for creating this useful resource. Im quite new to Github and I'm finding it quite difficult to set up my page. I was wondering if you have any videos/ step by step guides on how to do this? 

I really want to be able to use this to help publish my website. Here is a sample page.
https://remedymedicine.notion.site/General-Medicine-and-General-Surgery-b11707d1b5d6449180b9b6dc00eb796c

Thank you once again!

**Comment by transitive-bullshit** [comment]
Thanks for the kind words 🙏 

Honestly, this repo is geared more for folks who want more control over the code.

If you want a plug & play solution, I'd highly recommend using [super.so](https://s.super.so/x).

**Comment by Bandos1000** [comment]
@remedy123ate , I used this article: https://www.jarrodkane.com/notion-blog-made-with-nextjs
By far the easiest tutorial for this repo that I've seen. He really broke it down step by step. I was able to get it working by my second try. I used a fresh Github and Vercel instance just to ensure fewer errors. Hope it helps!
